plugins/janus_sip.c

```diff
@@ -760,45 +760,32 @@ int janus_sip_init(janus_callbacks *callback, const char *config_path) {
 		gboolean local_ip_set = FALSE;
```

Comment:
It's hard to be sure when looking at the diff, but if you do initialise the `local_ip` pointer to NULL, you may no longer need this local variable?